js_modules/dagster-ui/packages/ui-core/src/runs/RunActionsMenu.tsx

```diff
@@ -55,7 +55,9 @@ interface Props {
 }
 
 export const RunActionsMenu = React.memo(({run, onAddTag, additionalActionsForRun}: Props) => {
-  const {RunMetricsDialog} = React.useContext(InjectedComponentContext);
+  const {
+    components: {RunMetricsDialog},
+  } = React.useContext(InjectedComponentContext);
```

Comment:
Think we need the sub-object to separate hooks and components? I think it's fine, just find this less pretty